travis-run.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
+
+make rebuild && \
+  sudo make install DESTDIR=/Library/RubyMotion/BridgeSupport3 && \
```

Comment:
Sorry I don't have a plan to depend to RubyMotion

Reply:
Ah, it was just the location Amir gave me for putting BridgeSupport3, we could also just make DESTDIR be in a build directory. There's no dependencies on RubyMotion as far as I am aware.